syncd/FlexCounter.cpp

```diff
@@ -1367,6 +1417,54 @@ void FlexCounter::collectPriorityGroupAttrs(
     }
 }
 
+void FlexCounter::collectMACsecSAAttrs(
+        _In_ swss::Table &countersTable)
+{
+    SWSS_LOG_ENTER();
+
+    // Collect attrs for every registered MACsec SA
+    for (const auto &kv: m_macsecSAAttrIdsMap)
+    {
+        const auto &macsecSAVid = kv.first;
+        const auto &macsecSARid = kv.second->m_macsecSAId;
+        const auto &macsecSAAttrIds = kv.second->m_macsecSAAttrIds;
```

Comment:
this is interesting, so actual id's are here as a numbers ? not as stats name strings ? i dont know how other counters are passed, but i was under impressions that those id's were passed from OA as actual stats strings and not as id number, but i maybe mistaken